src/modules/Instabug.ts

```diff
@@ -196,15 +196,16 @@ export const setString = (key: StringKey, string: string) => {
 };
 
 /**
- * Sets the default value of the user's email and hides the email field from the reporting UI
+ * Sets the default value of the user's email and ID and hides the email field from the reporting UI
  * and set the user's name to be included with all reports.
  * It also reset the chats on device to that email and removes user attributes,
  * user data and completed surveys.
  * @param email Email address to be set as the user's email.
  * @param name Name of the user to be set.
+ * @param id optional ID of the user to be set.
```

Comment:
We can change `optional` to the JSDoc syntax of optional parameters `[id]`.

```suggestion
 * @param [id] ID of the user to be set.
```